packages/components/time-picker/src/common/picker.vue

```diff
@@ -313,7 +318,9 @@ const emitInput = (input: SingleOrRange<DateModelType> | null) => {
     } else if (input) {
       formatted = formatter(input, props.valueFormat, lang.value)
     }
-    emit(UPDATE_MODEL_EVENT, input ? formatted : input, lang.value)
+
+    const date = input ? formatted : input
+    emit(UPDATE_MODEL_EVENT, date, lang.value)
```

Comment:
There is no need to modify this variable. There is no place to reuse it, so there is no need.